packages/vscode-extension/package.nls.json

```diff
@@ -157,6 +158,28 @@
     "teamstoolkit.commandsTreeViewProvider.manageCollaboratorTitle": "Manage Collaborator",
     "teamstoolkit.commandsTreeViewProvider.manageCollaboratorDescription": "Manage M365 Teams App (with Microsoft Entra app) Collaborators",
     "teamstoolkit.commandsTreeViewProvider.addWebpartTitle": "Add SPFx Web Part",
+    "teamstoolkit.commandsTreeViewProvider.publishAppSourceTitle": "Publish to AppSource",
+    "teamstoolkit.commandsTreeViewProvider.publishAppSourceDescription": "Link to the wiki about how to publish the add-in to AppSource",
+    "teamstoolkit.commandsTreeViewProvider.createOfficeAddInTitle": "Create a new project",
+    "teamstoolkit.commandsTreeViewProvider.createOfficeAddInDescription": "Create a new add in project of Word Excel Or PowerPoint",
```

Comment:
It should be - Create a new add-in project of Word Excel or Powerpoint

Reply:
fixed